src/workers/notifications/postDeleted.ts

```diff
@@ -0,0 +1,47 @@
+import { messageToJson, Worker } from '../worker';
+import { Notification, Post } from '../../entity';
+import { ChangeObject } from '../../types';
+
+interface Data {
+  post: ChangeObject<Post>;
+}
+
+const worker: Worker = {
+  subscription: 'api.post-deleted-notification-cleanup',
+  handler: async (message, con, logger) => {
+    const data: Data = messageToJson(message);
+    const { post } = data;
+
+    const databasePost = await con
+      .getRepository(Post)
+      .findOneBy({ id: post?.id });
+    if (!databasePost || !databasePost?.deleted) {
+      return logger.error(
```

Comment:
Let's change the logger level to something else, otherwise we will get false alerts